assets/app.scss

```diff
@@ -0,0 +1,185 @@
+@import 'reset';
+
+$fa-font-path: "//netdna.bootstrapcdn.com/font-awesome/4.3.0/fonts" !default; // for referencing Bootstrap CDN font files directly
+
+@import 'font-awesome';
+@import 'bourbon';
+@import 'neat';
+
+@import "sassmatic";
+
+$black: #191919;
+$lightgrey: #464346;
+$white: #fff;
+$orange: #EB393B;
+$font: 'Montserrat', sans-serif;
+
+/* apply a natural box layout model to all elements, but allowing components to change */
+html {
+  box-sizing: border-box;
+  font-family: $font;
+  background-color: $black;
+  color: $white;
+  padding: 1em 1em 2em 2em;
+}
+*, *:before, *:after {
+  box-sizing: inherit;
+}
+.icons {
+	color: $lightgrey;
+}
+.name1 {
+	padding-left: 3em;
+}
+.name2 {
+	color: $lightgrey;
+}
+.links {
+	text-decoration: none;
+	color: $lightgrey;
+	float: right;
+}
+.links a:nth-of-type(2) {
+	color: $orange;
+}
+.links a:hover {
+	color: $orange;
+}
+.logo {
+	position: relative;
+	text-align: center;
+	width:100%;
+}
+.chevron {
+	font-size: 2em;
+	color: $orange;
+	position: absolute;
+	top:50%;
+	left: 49%;
+}
+.logo img {
+	@extend %filter-casablanca;
+	width: 9em;
+	border-radius: 5em;
+}
+.title {
+	position: absolute;
+	top:90%;
+	left: 46.8%;
+}
+
+.row {
+	padding-top: 3em;
+	@include outer-container(1140px);
+	padding-bottom: 3em;
+}
+
+.row2 {
+	text-align: center;
+}
+
+.row2 img {
+	@extend %filter-noire;
+}
+.row2 img:hover {
+	@extend %filter-nikki;
+}
+
+.description {
+	li:nth-of-type(1) {
+	color: $orange;
+	font-weight: bold;
+}
+	line-height: 2em;
+	display: inline-block;
+	vertical-align: top;
+	padding-left: 1em;
+	text-align: left;
+}
+.phototitle {
+	color: lighten($lightgrey,30%);
+	font-weight: bold;
+	font-size: 1.3em;
+	display: block;
+	position: relative;
+	left: 10em;
+	padding-top: .8em;
+}
+.date {
+	color: $lightgrey;
+	font-size: .8em;
+	position: relative;
+	left: 16.4em;
+}
+.description li {
+	color: lighten($lightgrey,30%)
+}
+.description [class*="fa"] {
+	color: $lightgrey;
+}
+
+.bottom-text {
+	padding-bottom: 2em;
+}
+
+.row-bottom {
+	@include outer-container(1140px);
+	text-align: center;
+	padding-top: 5em;
+}
+
+.row-bottom img {
+	@extend %filter-noire;
+	width: 25em;
+}
+
+.row-bottom img:hover {
+	@extend %filter-nikki;
+}
+
+.bottom-t {
+	@include span-columns(4);
+
+	&:nth-of-type(odd) {
+		@include shift(2);
+	}
+
+	img {
+		width: 100%;
+	}
+}
+
+.bottom-t a {
+	display: block;
+}
+
+.bottom-t a:hover {
+	color: $orange;
+}
+
+@media (max-width: 1024px) {
+div {
+	background-color: $black;
```

Comment:
![waaaaa](http://gifrific.com/wp-content/uploads/2015/02/Despicable-Me-Minion-Saying-What.gif)

Why is this a thing?

I see that it hides the links but SUPER HACK!!! Do something with `.links` and maybe `display` or `visibility`.